src/storage/page_buffer.c

```diff
@@ -1807,6 +1851,8 @@ pgbuf_fix_release (THREAD_ENTRY * thread_p, const VPID * vpid, PAGE_FETCH_MODE f
       pgbuf_hit = true;
 #endif /* ENABLE_SYSTEMTAP */
 
+      pgbuf_Pool.show_status.now.num_hit++;
```

Comment:
It seems that a race condition for the global status variable can occur. I couldn't find any latch mechanism to prevent it.